test/unit/gateways/authorize_net_test.rb

```diff
@@ -626,6 +626,18 @@ def test_truncation
     end.respond_with(successful_purchase_response)
   end
 
+  def test_invalid_cvv
+    invalid_cvvs = ['47', '12345', '']
+    invalid_cvvs.each do |cvv|
+      card = credit_card(@credit_card.number, { verification_value: cvv })
+      stub_comms do
+        @gateway.purchase(@amount, card)
+      end.check_request do |endpoint, data, headers|
+        parse(data) { |doc| assert_nil doc.at_xpath('//cardCode') }
```

Comment:
Small little nit - might just checking that the data doesn't include `cardCode` be sufficient?  Or is that included elsewhere in data?